### ISSUE TITLE
Relationship between immune checkpoint proteins, tumour microenvironment characteristics, and prognosis in primary operable colorectal cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#20]
## TL;DR

The tumour microenvironment is an important factor for colorectal cancer prognosis, affecting the patient's immune response. Immune checkpoints, which regulate the immune functions of lymphocytes, may provide prognostic power. This study aimed to investigate the prognostic value of the immune checkpoints TIM‐3, LAG‐3 and PD‐1 in patients with stage I–III colorectal cancer.

### Paper Link

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7869939/

### Author/Institution

Sara SF Al‐Badran (University of Glasgow)

## Overview

The tumour microenvironment is an important factor for colorectal cancer prognosis, affecting the patient's immune response. Immune checkpoints, which regulate the immune functions of lymphocytes, may provide prognostic power. This study aimed to investigate the prognostic value of the immune checkpoints TIM‐3, LAG‐3 and PD‐1 in patients with stage I–III colorectal cancer.

## Methods

colorectal cancer survival analysis

## Cite

Al-Badran SS, Grant L, Campo MV, Inthagard J, Pennel K, Quinn J, Konanahalli P, Hayman L, Horgan PG, McMillan DC, Roxburgh CS, Roseweir A, Park JH, Edwards J. Relationship between immune checkpoint proteins, tumour microenvironment characteristics, and prognosis in primary operable colorectal cancer. J Pathol Clin Res. 2021 Mar;7(2):121-134. doi: 10.1002/cjp2.193. Epub 2020 Dec 18. PMID: 33338327; PMCID: PMC7869939.
